src/pages/Dashboard/Modals/SupplyWithdraw/index.tsx

```diff
@@ -201,6 +187,24 @@ export const SupplyWithdrawUi: React.FC<ISupplyWithdrawUiProps & ISupplyWithdraw
     },
   ];
 
+  // Prevent user from being able to supply UST or LUNA
+  if (assetId !== 'ust' && assetId !== 'luna') {
```

Comment:
I took a slightly different approach here - https://github.com/VenusProtocol/venus-protocol-interface/pull/514

We don't have to take it but the idea was to start setting up feature flags so in the future we don't only have to add tokens to the disabled token list we don't have to change other application code too

Reply:
Sorry for not seeing you opened a PR for that already.

I updated this PR to follow the same logic as yours :)